cmd/clusterctl/client/describe.go

```diff
@@ -45,9 +45,9 @@ type DescribeClusterOptions struct {
 	// or it has the same Status, Severity and Reason of the parent's object ready condition (it is an echo)
 	DisableNoEcho bool
 
-	// DisableGrouping disable grouping machines objects in case the ready condition
-	// has the same Status, Severity and Reason
-	DisableGrouping bool
+	// Grouping groups machines objects in case the ready conditions
+	// have the same Status, Severity and Reason
+	Grouping bool
```

Comment:
Wondering if we need a newDescribeClusterOptions that returns the struct with Grouping defaulted to true, but let's do this only if provide advantages down the line (e.g when implementing tests)

Reply:
Same applies to DiscoverOptions too